vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/ui/GenericElementSteps.java

```diff
@@ -77,4 +95,52 @@ public void assertElementsNumberInState(State state, Locator locator, Comparison
         assertElementsNumber(locator, comparisonRule, quantity)
                 .forEach(e -> baseValidations.assertElementState(description, state, e));
     }
+
+    /**
+     * Verifies if the elements located by the <b>locator</b> are sorted in <b>sortingOrder</b> order by their text
+     * @param locator locator to locate elements
+     * @param sortingOrder sorting order, can be either <b>ASCENDING</b> or <b>DESCENDING</b>
+     */
+    @Then("elements located `$locator` are sorted by text in $sortingOrder order")
+    public void areElementSorted(Locator locator, StringSortingOrder sortingOrder)
+    {
+        ComparisonRule rule = ComparisonRule.GREATER_THAN_OR_EQUAL_TO;
+        int requiredNumber = 2;
+
+        String order = sortingOrder.name().toLowerCase();
+        List<WebElement> elements = baseValidations.assertNumberOfElementsFound(
+                format("The elements to check the %s order", order), locator,
+                requiredNumber, rule);
+
+        Matcher<Integer> requiredSizeMatcher = rule.getComparisonRule(requiredNumber);
+        if (requiredSizeMatcher.matches(elements.size()))
+        {
+            IntStream.range(0, elements.size())
+                     .mapToObj(index -> entry(index, getText(elements.get(index))))
```

Comment:
simple loop is better here

Reply:
why? if it's because of separate method for getting text I can use Pair which is not null safe